_rules/additional-content-triggered-on-hover-is-hoverable-ep1s13.md

```diff
@@ -0,0 +1,107 @@
+---
+id: ep1s13
+name: Additional content triggered on hover is hoverable
+rule_type: atomic
+description: |
+  This rule checks that any additional content that was triggered by hovering an element remains available to the user while the mouse pointer is over that content
+accessibility_requirements:
+  wcag21:1.4.13: # Content on Hover or Focus (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Mouse pointer
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any element that when [hovered][] causes [visible changes in content][].
+
+## Expectation
+
+There are no [visible changes in content][] while the target element is [hovered][] or the mouse pointer remains inside a rectangle defined by the following coordinates:
+
+- (**top**:) the lowest value of the y coordinate of any pixel affected by the [visible changes in content][]; and
+- (**left**:) the lowest value of the x coordinate of any pixel affected by the [visible changes in content][]; and
+- (**bottom**:) the highest value of the y coordinate of any pixel affected by the [visible changes in content][]; and
+- (**right**:) the highest value of the x coordinate of any pixel affected by the [visible changes in content][].
+
+## Assumptions
+
+- The user does not dismiss the tooltip causing the changes in content by pressing a key on the keyboard. In this instance the rule will fail while [success criterion 1.4.13: Content on Hover or Focus][sc1.4.13] might be satisfied.
```

Comment:
comment on "... the tooltip causing the changes in content" changes caused by hover does not change the original content which is already presented on a page but displays additional content instead.

Reply:
Once again: the definition of visible changes in content refers to changes in the pixel of the page (irrespectively of which elements are added or removed or of having their properties changed).
I went with pixels being changed because that is something objective and avoids the "confusion" that comes from trying to identify the origin of the change.

Reply:
Agree.